src/steps/02-EvaluationCriteria/EvalPlan/components/CreateEvalPlanSlideOut.vue

```diff
@@ -9,6 +9,24 @@
       criteria for selecting a source based on submission of a 
       quote/proposal/white paper.
     </p>
+    <h3 class="mb-5">What factors should be considered when deciding whether 
```

Comment:
I cannot figure out where this code is being used.  It should be in the at the red mark of the screencap.

image.png